### PR TITLE
rendering without styling formatting or data

### PR DESCRIPTION
**Pull request by bmblumenfeld**

### CODE REVIEW

File: description/client/components/App.jsx [hunk 1]
```diff
@@ -1,14 +1,34 @@
...
   constructor(props) {
     super(props);
+
+    this.state = {
+      course: {}
+    };
+  }
+
+  componentDidMount() {
+    axios.get("/description").then(response => {
```
Comment (jsonbaozi) [inline description/client/components/App.jsx]: Single quotes preferred

**Comment by jsonbaozi** [comment]
Content looks good. A few style tips:

Single quotes in javascript, double quotes in HTML / jsx "HTML". 
Try to keep your indentation consistent
For functional stateless components try using this structure: https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless
```
// good
function Listing({ hello }) {
  return <div>{hello}</div>;
}
```

**Comment by Camnix** [comment]
I would delete this commit and only push your changes, not the ones that include my branch folder in description

**Review by Camnix — APPROVED** [review]
looks good

**Pull request merged**